pom.xml

```diff
@@ -55,7 +55,7 @@
 
     <properties>
         <!--todo The classification is too confusing, reclassify by type-->
-        <revision>2.3.5-SNAPSHOT</revision>
+        <revision>2.3.4</revision>
```

Comment:
revert？

Reply:
Ok, I  forgot to modify this file because on my local windows environment  setting "2.3.5-SNAPSHOT" will cause build fails.

Reply:
Fixed.